lib/SIL/IR/SILType.cpp

```diff
@@ -1247,7 +1247,12 @@ SILType SILType::removingAnyMoveOnlyWrapping(const SILFunction *fn) {
 }
 
 bool SILType::isSendable(SILFunction *fn) const {
-  return getASTType()->isSendableType();
+  switch (getCategory()) {
```

Comment:
This is going to change the behavior of other existing callers in the compiler. I'm also not convinced this is necessary. Whether a type is loadable or address-only is an implementation detail of the type and should not be observable in the language semantics in any way.